### DISCUSSION TITLE
Adding a redirect uri for implicit oauth

### DISCUSSION THREAD

**Discussion by jimmywyatt** [hoppscotch/hoppscotch#4306] [Q&A]
I can't see a way to set the redirect uri when using OAuth 2.0 authentication, am I missing something?

**Comment by fredericsimard** [comment]
I have the same issue, can't get it to work with my Google Cloud project, even though I added the callback URL to the credentials section of the project.

Are you running it locally or via the cloud?

**Reply by jimmywyatt** [reply]
Locally for me

**Comment by allen-liaoo** [comment]
please add this!
